src/include/migraphx/module.hpp

```diff
@@ -245,6 +245,21 @@ struct MIGRAPHX_EXPORT module
                                      const std::vector<instruction_ref>& splits1,
                                      const std::vector<instruction_ref>& splits2) const;
 
+    // Fuse the instruction into the module by inserting the instructions and
```

Comment:
Is it obvious to people who know the codebase what the inputs and outputs of these methods are?